src/auth/tenant.ts

```diff
@@ -46,11 +77,12 @@ export interface TenantServerResponse {
 /**
  * Tenant class that defines a Firebase Auth tenant.
  */
-export class Tenant implements TenantInterface {
+export class Tenant {
+
   public readonly tenantId: string;
   public readonly displayName?: string;
-  public readonly emailSignInConfig?: EmailSignInConfig;
-  public readonly multiFactorConfig?: MultiFactorAuthConfig;
+  private readonly _emailSignInConfig?: EmailSignInConfig;
```

Comment:
You seem to use trailing underscore for private variables above, eg. `app_`. It would be a good idea to be consistent (either trailing or leading).

Reply:
Thanks. Good point and fixed.